src/redux/categories/categories.js

```diff
@@ -0,0 +1,22 @@
+// Actions
+const CHECK_STATUS = 'bookStore/categories/CHECK_STATUS';
+
+// Reducer
+const reducer = (state = [], action = {}) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under construction';
+
+    default: return state;
+  }
+};
+
+// Action Creators
+const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
+
+export {
+  reducer as default,
+  checkStatus,
+};
```

Comment:
at this stage you should be able to display the status and use your redux code by dispatching the status when click on the button  right now your application it's now working as expected so make sure to add the code to implement that